### ISSUE TITLE
How up to date is this documentation?

### ISSUE THREAD

**Issue by DanDiplo** [lever/postings-api#34]
I've been asked to look into using the Lever API for a client who will (but currently hasn't) signed with Lever.   I've received a copy of the document **Lever Developer Documentation** dated 9/6/2018 which includes a link to this repo. However, I can see this documentation references `v0` of the API whereas the PDF document I have always refers to `v1` ie. the endpoint is https://api.lever.co/v1 

It seems these two versions of the API are quite different in many ways:

- `V1` requires authentication to get job postings, where as `V0` doesn't
- `V1` is able to return published, internal, closed and draft postings
- The fields returned in a posting in `V1` are quite different from `V0`

So I'm a little confused as to what to use. It seems `V1` is the most up-to-date version, but this isn't mentioned anywhere in this repo documentation and there are no examples.

So could you clarify if someone was to create a new integration  with Lever which version of the API should they be using? And if it is `V1` then will you be updating this repo to provide information on this and, ideally, a test API key people can use to test with? Thanks.

**Comment by plcjunqueira** [comment]
@DanDiplo did you find a solution for your issue?

**Comment by DanDiplo** [comment]
@plcjunqueira No, I've not heard anything back, which is a little concerning given this is an API for a widely used commercial project.

**Comment by distracteddev** [comment]
Hi there and sorry for the late reply. 

The two APIs serve separate purposes and v1 should not be thought of as a successor to v0. 

The Postings API (v0) is meant to provide all functionality needed to create a custom public job site and only includes publicly available data. 

The Hire API (v1) is meant to serve all other API integration needs and offers an expansive set of endpoints to support a variety of integration types. This API includes access to private data and thus requires authentication. 

This Hire API documentation is not currently publicly accessible and requires an existing Lever account to access. This private documentation lives at developer.lever.co

Although I am more than happy to continue providing assistance here, our support channel (support@lever.co)  will guarantee you the fastest response. 

The tl;dr is that you should use the Postings API (v0) if you are creating a public job site integration. For all other integrations, you’ll want to use the Hire API (v1).
